packages/pigeon/platform_tests/alternate_language_test_plugin/android/src/test/java/com/example/alternate_language_test_plugin/AsyncTest.java

```diff
@@ -61,7 +61,7 @@ public void asyncSuccess() {
             (bytes) -> {
               bytes.rewind();
               @SuppressWarnings("unchecked")
-              ArrayList wrapped = (ArrayList) codec.decodeMessage(bytes);
```

Comment:
I had to fix a bunch of raw type warnings in our tests, and suppress a bunch of unchecked casts for most of the same lines.